### ISSUE TITLE
transparentTileErrorColor - unable to find property error

### ISSUE THREAD

**Issue by ipasic** [mapfish/mapfish-print#650]
Hi, I want to print some custom tiled layers and it works fine until I define "transparentTileErrorColor" property in my configuration file (config.yaml).
As far as I understood, that should be string which defines what color should be used for tiles in error on transparent layers. But as soon as I add this property and request map report from I get the error

> "Unable to find property 'transparentTileErrorColor' on class: org.mapfish.print.config.Configuration"

My config.yaml file looks like this

```
transparentTileErrorColor: rgba(0,0,0,0.3)
templates:
  A4 landscape: !template
    reportTemplate: report.jrxml
    attributes:
      title: !string {}
      map: !map
        maxDpi: 400
        width: 780
        height: 450
    processors:
    - !reportBuilder
      directory: '.'
    - !createMap {}
```

Can somebody explain how should i set this property correctly?

**Comment by pvalsecc** [comment]
Like you do. But this is available only since 3.10.2 (was broken before). What version do you use?

**Comment by ipasic** [comment]
Ah ok, i was with Mapfish 3.9.0, I will try then with newest version, thanks
